### PR TITLE
Fixed path in qc script, removed unwanted symbols from ensg mapping file

### PR DESCRIPTION
**Pull request by sangeetashukla**
### Purpose/implementation Section
Updated gene_match module to reflect correct path for the data files, and fix the `ensg-hugo-pmtl-mapping.tsv` file to filter out `ensg_id = Symbol_Not_Found` and `gene_symbol = NA`

#### What was your approach?

- Remove version (v10) specification from the `qc_ensg_hugo_pmtl_mapping.Rmd` script for reading in data files
- Add a filter to remove unwanted ensg_id and gene_symbol before writing to `ensg-hugo-pmtl-mapping.tsv` in the `add_pmtl_ens_hugo.Rmd` script

#### What GitHub issue does your pull request address?
[Issue 231](https://github.com/PediatricOpenTargets/ticket-tracker/issues/231)

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.
Please test if the scripts run as expected with symlink set up to read data files.

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes

#### What types of results are included (e.g., table, figure)?
`ensg-hugo-pmtl-mapping.tsv`

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/gene_match/qc_ensg_hugo_pmtl_mapping.Rmd [hunk 1]
```diff
@@ -14,7 +14,7 @@ suppressPackageStartupMessages({
 ## Read ensg-hugo-pmtl-mapping.tsv and OpenPedCan SNV, CNV, TPM, and fusion data
 
 ```{r read_open_ped_can_data}
-open_ped_can_data_path <- '~/OpenPedCan-analysis/data/v10'
+open_ped_can_data_path <- '~/OpenPedCan-analysis/data/'
```
Comment (ewafula) [inline analyses/gene_match/qc_ensg_hugo_pmtl_mapping.Rmd]: Change to: 
`open_ped_can_data_path <- '../../data'`

**Review by ewafula — CHANGES_REQUESTED** [review]
@sangeetashukla, please change to the `OpenPedCan-analysis/data` directory to be relative to the `qc_ensg_hugo_pmtl_mapping.Rmd` location as I have shown in the code comment.  Not everyone running the module will have  the `OpenPedCan-analysis` local cloned repo in their home directory (`~/`). Otherwise, the results are ok.  The `Symbol_Not_Found` and `NA` values  in `gene_symbol` and `ensg_id` column respectively have been filtered out of the  `results/ensg-hugo-pmtl-mapping.tsv` file.

```
Quitting from lines 17-46 (qc_ensg_hugo_pmtl_mapping.Rmd)
Error: ‘~/OpenPedCan-analysis/data//snv-consensus-plus-hotspots.maf.tsv.gz’ does not exist.

Execution halted
```

**Comment by sangeetashukla** [comment]
> @sangeetashukla, please change to the `OpenPedCan-analysis/data` directory to be relative to the `qc_ensg_hugo_pmtl_mapping.Rmd` location as I have shown in the code comment. Not everyone running the module will have the `OpenPedCan-analysis` local cloned repo in their home directory (`~/`). Otherwise, the results are ok. The `Symbol_Not_Found` and `NA` values in `gene_symbol` and `ensg_id` column respectively have been filtered out of the `results/ensg-hugo-pmtl-mapping.tsv` file.
> 
> ```
> Quitting from lines 17-46 (qc_ensg_hugo_pmtl_mapping.Rmd)
> Error: ‘~/OpenPedCan-analysis/data//snv-consensus-plus-hotspots.maf.tsv.gz’ does not exist.
> 
> Execution halted
> ```

Thank you @ewafula  for catching that. I have updated the file path appropriately and tested the script.

File: analyses/gene_match/qc_ensg_hugo_pmtl_mapping.Rmd [hunk 2]
```diff
@@ -14,7 +14,7 @@ suppressPackageStartupMessages({
 ## Read ensg-hugo-pmtl-mapping.tsv and OpenPedCan SNV, CNV, TPM, and fusion data
 
 ```{r read_open_ped_can_data}
-open_ped_can_data_path <- '~/OpenPedCan-analysis/data/'
+open_ped_can_data_path <- '../../data/'
```
Comment (ewafula) [inline analyses/gene_match/qc_ensg_hugo_pmtl_mapping.Rmd]: @sangeetashukla, trailing backslash is unnecessary. R will add when joining to create a path. Will still work but you will have double backslashes as shown in the previous error message that I posted.
 set to this: `open_ped_can_data_path <- '../../data'`

**Review by ewafula — CHANGES_REQUESTED** [review]

**Comment by sangeetashukla** [comment]
> Thanks @sangeetashukla for preparing this! Everything looks good to me now (thanks @ewafula for the detailed review). @sangeetashukla, could you please do a dev merge and I think this should be ready for merge.

@runjin326 Done

**Pull request merged**